packages/cli-platform-apple/src/commands/runCommand/createRun.ts

```diff
@@ -310,15 +310,17 @@ const createRun =
     }
 
     if (args.udid) {
-      const device = devices.find((d) => d.udid === args.udid);
+      let device = devices.find((d) => d.udid === args.udid);
       if (!device) {
-        return logger.error(
+        // We may be able to recover from this for macCatalyst app starts,
+        // but the general case is this is a problem and device list will help
+        return logger.warn(
```

Comment:
"warn" now, not "error" because it may be recoverable (that's my criteria for error vs warn)
But it is still serious and the device list could help people